src/renderer/hooks/useProcessQr.ts

```diff
@@ -47,43 +67,110 @@ export default function useProcessQR() {
     [openAlertDialog]
   )
 
+  // Users can enter the "Instant Onboarding" flow by scanning a DCACCOUNT,
+  // DC_ASK_VERIFYGROUP, or DC_ASK_VERIFYCONTACT code which essentially creates
+  // a new "chatmail" profile for them and connects them with the regarding
+  // contact or group if given.
+  //
+  // We ask the user if they really want to proceed with this action and
+  // accordingly prepare the onboarding.
+  const startInstantOnboarding = useCallback(
+    async (accountId: number, qrWithUrl: QrWithUrl) => {
+      const { qr } = qrWithUrl
+      const numberOfAccounts = (await BackendRemote.rpc.getAllAccountIds())
+        .length
```

Comment:
we should rather check if the current account is new (unconfigured), otherwise the message is wrong when you have exactly one configured account and scan the code.